js/completionCalculations.js

```diff
@@ -45,8 +45,8 @@ define([
                 "assessmentTotal": assessmentComponentsTotal
             };
 
-            if (contentObjectModel.get("_pageLevelProgress") && contentObjectModel.get("_pageLevelProgress")._showPageCompletion !== false 
-                && Adapt.course.get("_pageLevelProgress") && Adapt.course.get("_pageLevelProgress")._showPageCompletion !== false) {
+            if (contentObjectModel.get("_pageLevelProgress") && contentObjectModel.get("_pageLevelProgress")._showPageCompletion !== false &&
```

Comment:
split up into variables?
```
var showPageCompletionOnCO = contentObjectModel.get("_pageLevelProgress") && contentObjectModel.get("_pageLevelProgress")._showPageCompletion !== false;
var showPageCompletionOnCourse = Adapt.course.get("_pageLevelProgress") && Adapt.course.get("_pageLevelProgress")._showPageCompletion !== false;

if (showPageCompletionOnCO && showPageCompletionOnCourse) {
  // ...
}

```
not tested the logic and not sure about the var names, but it makes it easier to read

Reply:
I'll have a look in a bit..!